source/codegen/metadata/nifake_non_ivi/functions.py

```diff
@@ -164,7 +164,7 @@
             },
             {
                 'direction': 'out',
-                'get_last_error': 'get_last_error',
+                'get_last_error': 'deprecated',
```

Comment:
Note: For the MI drivers, we updated all the 'get_last_error' values to 'deprecated'. I did the same for all the non-MI drivers as well so that's what these 'get_last_error' updates are about.

This doesn't functionally matter because we don't look at the value for this field in code generation, we're just looking for the 'get_last_error' key to decide to mark the field deprecated in the proto file.